### DISCUSSION TITLE
Tokenization favorability in Cayman Islands

### DISCUSSION THREAD

**Discussion by ryanberckmans** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#27] [General]
To the Ministry of Financial Services and Cayman Islands Monetary Authority:

My name is Ryan Berckmans.

I'm a Cayman Islands permanent resident, property owner, high net worth individual, long-term Ethereum investor and insider, and occasional guest on Raoul Pal's Real Vision show.

I wanted to highlight the new Mountain Protocol [USDM](https://mountainprotocol.com/usdm) stablecoin as an example area of potential future growth for Cayman Islands.

The USDM stablecoin is similar to Circle's USDC or Tether's USDT, except unlike USDC and USDT, USDM pays out almost 100% of investment income on reserves directly to USDM token holders. USDM currently pays 5% annually.

When anybody holds a USDM balance of 1000 tokens, then one year later, they will automatically have approximately 1050 tokens in their wallet, with no action required on their part. It's as if the concept of a high-interest savings account has been built directly into the stablecoin itself.

On a technical basis, the interest on USDM is paid daily via a single onchain rebase operation, similar to Lido's stETH. Token holders do not need to "claim their yield"- it's automatic for all USDM holders. This automation creates simplicity and scalability that drives USDM's value proposition.

USDM is regulated in Bermuda, not Cayman Islands. I spoke with the founder of USDM, and Cayman was _not_ among his top preferred jurisdictions.

The rest of this letter explores why it may be advantageous for Cayman to become a preferred jurisdiction for stablecoins and global tokenization projects in general. I've tried to present a complete picture in brief; some of this may already be common knowledge.

**Stablecoins like USDM that automatically pay interest to their holders are the future of stablecoins.**
 
USDM and its future peers are vastly superior to offchain non-crypto cash and onchain stablecoins that pay no yield to users. For example, USDC keeps all investment income as operator revenue and pays 0% to token holders.

If the $100 billion in extant USDT were converted to USDM, the token holders would automatically enjoy an additional $5 billion yearly in investment income.

**Tokenization is the future of the global economy.**

In the coming decades, institutions worldwide will seek to **tokenize almost every asset, especially securities and fiat currencies**. 

Tokenization will become a huge force shaping the global economy.

Eventually, decades from now, most assets will be traded as tokens, and most asset ownership will occur via holding tokens.

**Cayman Islands can win tokenization growth.**

When an institution elects to create any new token, an initial task is to **select the jurisdiction where tokenization will occur**.

The choice of tokenization jurisdiction determines the legal foundation that makes the token valuable.

Crucially, an institution's choice of tokenization jurisdiction is typically mainly based on the **prospective jurisdiction's strength of regulatory frameworks**, property rights, and long-term economic, political, and regulatory **stability**, **not the jurisdiction's population or GDP**.

This makes **Cayman Islands a potential ideal jurisdiction to benefit from the coming windfall of global tokenization**.

**Cayman Islands is currently losing in tokenization.**

If the government may determine that winning a large share of global tokenization growth is a desirable outcome, then swift action must be taken to ensure that Cayman becomes a preferred jurisdiction for tokenization. Perhaps this revision to the VASP might be a vehicle for such change.

Empirically, **Cayman Islands is not presently a jurisdiction of choice for tokenization**. If nothing changes, my expectation is that this growth opportunity will mostly pass by Cayman and go to other countries.

Today, Cayman enjoys being a jurisdiction of choice for crypto funds and foundations. However, when institutions and new ventures seek to launch tokens, they are mostly picking, for example, Bermuda, Germany, etc.

I'm not an attorney or regulatory expert, and I can't speak to the specific regulation changes that may enable Cayman to become a preferred jurisdiction for tokenization. But I am a crypto and Ethereum expert, and I know Cayman is not winning these deals right now.

I'm reachable for comment here or in my Twitter [direct messages](https://twitter.com/ryanberckmans).

I would be happy to provide private commentary in person in Cayman, where I reside nearly year-round with my wife and children.

I'd also be happy to facilitate connections to industry tokenization experts, who would be able to provide more specific guidance on how Cayman Islands may become a preferred jurisdiction for tokenization.

**Comment by FinalTwist** [comment]
from what I understand tokenization (issuance of new tokens) isn't something that is tackled with this legislative change.  I guess what would be helpful is to know exactly why cayman was not on the list of jurisdictions for this token, or what you think could do better in that space.

I'm hoping for a healthy discussion in this forum on the topic!

**Reply by Aaron-XReg** [reply]
First, I agree with Ryan that tokenization represents a tremendous opportunity for the Cayman Islands. Currently, there is no clear cut jurisdiction of choice for tokenization projects internationally (or in the Caribbean). Tokenization is fully on the TradFi radar (see Blackrock comments etc.) and the blend of TradFi institutional interest and blockchain technology could be a perfect niche for Cayman.

So how do we make Cayman the jurisdiction of choice for tokenized securities, tokenized RWA's etc.?

The first most crucial step is regulatory certainty. Industry needs to have a clear road map of the regulatory process for a tokenization issuance. Currently virtual asset issuers are required to become registered under the VASP Act but the issuance approval regime is not yet in force. It is likely that CIMA will develop rules and guidance for issuances, but in the interim, tokenization projects need clarity with respect to expectations and best practices for tokenization projects to receive registration approval from CIMA. 

Please contribute ideas on what the regulatory regime for tokenization in Cayman could look like while it is being fully developed.

If we can provide industry with a level of regulatory clarity, Cayman is well placed to become a international tokenization hub and I can assure you that there will be significant market interest.

**Comment by JupiterBlock** [comment]
One of the reasons why Cayman is not selected for tokenization is the requirement that any public sales be pre-approved. 
In the absence of a clear understanding of time-to-approval, projects will not wish to get stuck in the pipeline when timing is critical to go to market. There could be two good solutions to this: (1) a company biting the bullet to become a regulated token issuance platform under the VASP Act, with certain parameters and processes approved by CIMA so that projects issuing via this platform do not need to pre-register - this is capital-intensive and time-intensive to set up; and (2) the VASP Act changing to allow for public sales of tokens with notification of the  offering materials but without waiting for approval from CIMA. This is the solution adopted under MiCA in the EU: a **notification requirement** to competent authorities with the obligation to draw up, notify and publish a crypto-asset white paper subject to detailed requirements + liability for the information given. Also, the public offering to fewer than 150 persons per Member State and the public of crypto-assets whose total consideration does not exceed 1 million Euros over the last 12 months are exempted.

**Reply by ryanberckmans** [reply]
Thanks, Jupiter, for the insightful comment.

I'm familiar with many crypto projects that have sought to launch tokens. However, due to their competitive need to move quickly and with certainty, they are unable to enter into potentially open-ended application processes for a public token offering.

Your proposed solution of allowing public sales of tokens with required notification of the offering but without waiting for approval from CIMA would be attractive to the crypto industry.

Crypto projects can't afford to file an application and wait many months to see how it turns out. The industry is fast-paced and globally competitive. They must pick jurisdictions that provide short and specific timelines. The cost of compliance is also a factor, but it is secondary.

Additionally, there is potential for creative tokenization compliance cost structures to create synergies with other Cayman sectors. 

For example, in September, I met with Wyoming's Stable Token Commission. I learned that while Wyoming intends to make it easy and cheap to do business with its stablecoin and attendant banks, it will require participants to incorporate in Wyoming.

Cayman Islands could do something similar where tokenization compliance is cheap, with short and certain timelines to public offerings, but requires the entity doing the tokenization to incorporate in Cayman Islands and even to do other kinds of business here as well.

In short, if Cayman Islands may create a path to public tokenization offerings that is cheap, simple, and hinges only on a required notification of public token offering but not CIMA approval, and then tie this privilege into the responsibility of incorporating and doing business in Cayman Islands, then I can see the crypto industry finding this quite attractive.

**Comment by AGIAI** [comment]
Notification instead of permission makes sense - I would suggest that CIMA be given 15 days to object to the proposed offering; providing remedies.

**Comment by FinalTwist** [comment]
What about the suggestion to use a regulated VASP for the token issuance?  If a project uses and engages a regulated VASP, the project does not need to register directly with CIMA or seek approval from CIMA.  Would the crypto sector accept this type of indirect supervision?

**Reply by ryanberckmans** [reply]
Yes, I think the crypto sector would welcome a marketplace of Cayman VASPs from which to purchase low-friction tokenization services at reasonable cost.

Regarding tokenization favorability in Cayman Islands, to the criteria above I'd add

- The capability for non-traditional entities such as individuals, non-profits, or DAOs to work with VASPs to launch tokens with the same low level of friction enjoyed by corporate customers. Ie. avoid restricting availability of VASP services to only customers that are registered corporations.

- A clear, low-friction path with reasonable cost to imbue tokens with underlying property rights in the Cayman Islands. For example, a token might be backed by equities, real estate, collectibles, or even a service obligation, such as the right to redeem a token for a week at a Cayman resort. For any given token, the underlying property right should have a clear path to enshrinement in Cayman law (eg. hold US equities in a Cayman Trust and then legally connect the Cayman Trust to the token). As well, the law should stipulate that tokens are "bearer" assets ie. he who holds the token owns the claim on the underlying property- as opposed to having a separate official registry of token property owners held off the blockchain, which defeats the purpose of tokenization. Note that just because tokens are bearer assets does not make them incompatible with various KYC/AML regimes, including the requirement for a token holder register/KYC to redeem a token for the underlying property right.

**Comment by AGIAI** [comment]
Clarity over tokenized issuance of securities (shares) share would also be welcome.

In France for example they regulated this as a finance matter - you can choose to have your shares as a paper share certificate or electronic - the electronic version can be a security token. These tokens can be swapped for the paper certificate and visa versa. 

It would make sense to ensure that security tokens are not captured under VASP and are instead dealt with under the Company Law. 

We will see an increasing number of securities tokenized, particularly by large funds (KKR, Blackrock etc.) and Cayman should have clarity  how the fact that these are different to other forms of issuance.

**Reply by AGIAI** [reply]
These should then be treated like other securities (i.e. same rules as the SIBA legislation)

**Reply by Aaron-XReg** [reply]
I completely agree. Tokenized securities they should be dealt with under SIBA. Whether your evidence of investment is a paper certificate, a certificate in electronic form or a token should not trigger different regulatory treatment. As discussed above there is a huge opportunity for Cayman here and no policy basis for subjecting tokenized securities to a more onerous regime or regulatory uncertainty. It will be important to communicate the treatment of tokenized securities to industry through a clear amendment and guidance if necessary.

**Comment by Aaron-XReg** [comment]
The Cayman Islands was once a global leader in virtual asset (‘VA’) issuances. The VASP Act was designed to, among other objectives, ensure that the Cayman Islands was fully compliant with the Financial Action Task Force (‘FATF’) Recommendations for virtual assets. However, the VASP Act goes beyond the FATF Recommendations in terms of VA issuance, where the VASP Act introduced a registration requirement and issuance approval regime for VA issuers.  VA issuance in itself is not caught within the scope of the FATF Recommendations.  This inclusion has resulted in a migration of issuers to jurisdictions like the British Virgin Islands (‘BVI’), which have maintained consistency with the scope of the FATF Recommendations.

The VA issuance market has evolved significantly from the initial coin offering (ICO) boom in 2017. It now includes a diverse range of products such as stablecoins, tokenised securities, real-world assets, and tokens that serve in the governance of decentralised autonomous organisations. It is widely expected that VA issuances will become a cornerstone of the next generation of finance. To quote Blackrock’s CEO Larry Fink, “the next generation for markets, the next generation for securities, will be tokenisation of securities." 

The World Economic Forum estimates that 10% of Global GDP – including government and corporate bonds, private debt and equity, real estate, carbon credits, and longtail insurance – will be tokenized by 2030. Tokenization converts the rights of assets into blockchain-based digital tokens. This innovation promises instant transaction settlement, enhanced liquidity, evidence of ownership, asset automation, transparency and auditability.

The Cayman Islands has the opportunity to be a leader in the next generation of markets and create a more diversified and innovative economy. However, the current approach to the regulation of VA issuances could see much of this opportunity slip through its grasp. One such missed opportunity is a recently announced Blackrock tokenization project that has launched through a BVI structure. 
The time to demonstrate leadership in the virtual asset issuance space is now. Bold action and a deconstruction of the current approach will be necessary for the Cayman Islands to truly become a global leader. Therefore, we recommend that VA issuances be removed from the scope of the VASP Act until a suitable regulatory framework has been developed.

In the alternative, we recommend that the VASP Act be amended to create a separate category of authorisation for issuances that, at the initial stage, do not require registration but include due diligence requirements for directors, beneficial owners and notification requirements to CIMA when an issuance has taken place. This approach could be supplemented by an interim Statement of Principles or code of conduct for issuers.

With respect to tokenised securities, an interim approach could be to explicitly exclude them from the VASP Act and treat tokenised securities as securities under the Securities Investment Business Act (SIBA).

Adopting a staged and nuanced approach would offer greater clarity to the industry with internationally consistent, escalating regulatory safeguards, ensuring that VAs are issued prudently and do not expose the jurisdiction to reputational or economic risks. 

The Cayman Islands has long been the leading international fund issuance jurisdiction by adopting a pragmatic approach to regulation that allows industry to thrive in a safe and compliant manner. If this approach is brought to the VA issuance space, the jurisdiction can also count on being a leader in the next generation of markets.

**Reply by CEOBert** [reply]
Hear, hear!

**Comment by Cayman-Blockahain-Association** [comment]
Based on the round table discussion hosted by BACI to discuss the VASP amendments with our members they strongly agreed that Cayman should reconsider this point. 

Some of the suggestions raised above are very valid especially around tokenized securities and clarity around the treatments of these assets.
